bin/cml/runner.js

```diff
@@ -469,7 +469,7 @@ exports.builder = (yargs) =>
       },
       cloudGpu: {
         type: 'string',
-        choices: ['nogpu', 'k80', 'v100', 'tesla'],
+        choices: ['nogpu', 'k80', 'v100', 'tesla', null],
```

Comment:
Interesting that this is not working. I was as far as I remember. Maybe using ```undefined``` in the coerce function do the trick? If thats the case maybe is better than null explicitly which is going to be a difficult type to setup

Reply:
I tried changing the value and it seemed that if the result of the coerce function isn't in the choices array then it fails.
My very first patch was to set it to `'nogpu'` instead of null which worked fine.
I opted to add `null` as an option instead to preserve the truthy/falsy behavior in ternary statements.

Reply:
I'll try this and we can see.
```js
        choices: ['nogpu', 'k80', 'v100', 'tesla'],
        coerce: (val) => (val === 'nogpu' ? undefined : val),

```

Reply:
Thats what I suggested

Reply:
@DavidGOrtega It looks like `undefined` behaves as expected, Im going to do a full test in our gcp env and if it works as expected I'll update the PR.

Reply:
See also https://github.com/yargs/yargs/issues/756

Reply:
> @DavidGOrtega It looks like undefined behaves as expected, Im going to do a full test in our gcp env and if it works as expected I'll update the PR.

@DanielBarnes that was my intuition, since its allowing you to not set it, I would say that this worked also with null previously, but not sure... JS and its non "Good parts" 🙏

Reply:
> JS and its non "Good parts" 🙏
🤣